src/DSP.jl

```diff
@@ -2,14 +2,37 @@ __precompile__()
 
 module DSP
 
+# We want to be very sure we don't pull in Base names unless we're very sure we want them
+# This macro will be called in each submodule herein to do the appropriate imports
+macro importffts()
+    quote
+        using AbstractFFTs, FFTW
+        importall AbstractFFTs, FFTW
+        if VERSION >= v"0.7.0-DEV.585"
```

Comment:
I can just set the version cutoff to something way off, e.g. `v"1.0.0"`, then go back and change it. That at least ensures that the code path doesn't get hit in the interim.